### PR TITLE
Documentation update for local secondary index

### PR DESCRIPTION
**Pull request by jovana**

### CODE REVIEW

File: docs/indexes.rst [hunk 1]
```diff
@@ -94,9 +94,10 @@ Local secondary indexes are defined just like global ones, but they inherit from
         forum = UnicodeAttribute(hash_key=True)
         view = NumberAttribute(range_key=True)
 
-
-You must specify the same hash key on the local secondary index and the model. The range key can be any attribute.
-
+Every local secondary index must meet the following conditions:
+- The partition key (hash key) is the same as that of its base table.
+- the sort key (range key) consists of exactly one scalar attribute. The range key can be any attribute.
```
Comment (ikonst) [inline docs/indexes.rst:99]:
```suggestion
- The sort key (range key) consists of exactly one scalar attribute. The range key can be any attribute.
```

**Comment by ikonst** [comment]
Just to understand, is this documenting DynamoDB or a PynamoDB-specific behavior?

**Comment by jovana** [comment]
> Just to understand, is this documenting DynamoDB or a PynamoDB-specific behavior?

It's for both. I did not understand what I was doing wrong, following the documentation gives me not the solution. After I understand these conditions needs to apply, the LSI works. So that is why I have suggested adding this conditions.

**Review by ikonst — APPROVED** [review]

**Comment by ikonst** [comment]
Thanks!

**Pull request merged**